comfy/k_diffusion/sampling.py

```diff
@@ -996,5 +996,45 @@ def post_cfg_function(args):
         dt = sigma_down - sigmas[i]
         x = x + d * dt
         if sigmas[i + 1] > 0:
+
```

Comment:
You accidentally removed code from this sampler.

Reply:
ah, my bad; should be fixed now